### ISSUE TITLE
@@toStringTag

### ISSUE THREAD

**Issue by zloirock** [zenparsing/es-observable#91]
`@@toStringTag` definitely should be added to all instances from this proposal.

**Comment by domenic** [comment]
Nope. New things (like Map, Set, Promise, user-defined classes ... and Observable) don't get toStringTag. It's a legacy language feature added to preserve back-compat.

**Comment by zloirock** [comment]
> Nope. New things (like Map, Set, Promise, user-defined classes ... and Observable) don't get toStringTag.

Seriously? :)

```js
Map.prototype[Symbol.toStringTag]; // => "Map"
Object.getPrototypeOf(new Map().values())[Symbol.toStringTag]; // => "Map Iterator"
Promise.prototype[Symbol.toStringTag]; // => "Promise"
SIMD.Float32x4.prototype[Symbol.toStringTag]; // => "SIMD.Float32x4"
```

**Comment by domenic** [comment]
You're right, I was confusing this with another toString-related spec thing. Apologies.

**Comment by benjamingr** [comment]
Looks like it's defined for everything separately, http://www.ecma-international.org/ecma-262/6.0/#sec-promise.prototype-@@tostringtag I was thinking species :D

**Comment by jhusain** [comment]
Certainly seems like this should be done. Will try and get around to this prior to the next meeting.
